typescript/infra/src/config/agent.ts

```diff
@@ -74,6 +74,8 @@ interface ValidatorSet {
 interface Validator {
   address: string;
   checkpointSyncer: CheckpointSyncerConfig;
+  identifier?: string;
```

Comment:
Hmm identifiers weren't originally intended for validators that we wouldn't operate, but I guess this would work

Is the intent to configure in `config/${environment}/validators.ts` a readonly validator whose `identifier` is created by a call to this function https://github.com/abacus-network/abacus-monorepo/blob/main/typescript/infra/src/agents/agent.ts#L177 where I guess the context is used as a way to identify the name of the operator? e.g. for the mainnet ZeePrime validator on eth `keyIdentifier(true, 'mainnet', 'zeeprime', 'validator', 'ethereum')`?